1.8/administration/banning

```diff
@@ -18,6 +18,26 @@ To ban an IP address, just enter the IP address, or range of IP addresses using
 
 You can later delete any banned IP address by simply selecting the delete icon on the right hand side of the page for the corresponding IP address.
 
+## Banned Accounts
+
+The banning section is located under the Users & Groups tab and it allows you to ban registered users.
+
+#### Banning Accounts
+
+To ban registered user you need to fill following fields:
```

Comment:
This should be:
To ban a registered user, you need to fill out the following fields: